### PR TITLE
chore: update components in /components to new grid except header

### PR DESCRIPTION
**Pull request by paulpopus**

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -262,15 +261,15 @@ export const Footer: React.FC<FooterType> = props => {
...
-            <Cell cols={4} colsM={8}>
+            <div className={['cols-6 cols-m-8'].filter(Boolean).join(' ')}>
               <p
                 className={classes.copyright}
               >{`Copyright ${new Date().getFullYear()} Payload CMS, Inc.`}</p>
-            </Cell>
+            </div>
 
-            <Cell cols={2} colsM={8} className={classes.themeCell}>
+            <div className={[classes.themeCell, 'cols-2 cols-m-8'].filter(Boolean).join(' ')}>
```
Comment (PatrikKozak) [inline src/components/Footer/index.tsx:272]: @paulpopus Are we sure this should remain `cols-2` even with the new `column` count?

File: src/components/Hero/ContentMedia/index.tsx [hunk 2]
```diff
@@ -19,23 +18,34 @@ export const ContentMediaHero: React.FC<
 > = ({ richText, media, mediaWidth, breadcrumbs, links }) => {
   return (
     <Gutter>
-      <Grid className={mediaWidth === 'wide' ? classes.wideGrid : classes.grid}>
-        <Cell start={1} cols={mediaWidth === 'wide' ? 5 : 7} colsM={8}>
+      <div
+        className={[mediaWidth === 'wide' ? classes.wideGrid : classes.grid, 'grid']
```
Comment (PatrikKozak) [inline src/components/Hero/ContentMedia/index.tsx:22]: @paulpopus Will need to switch the class name of `classes.grid` to `classes.normalGrid` or similar - that way `classes.grid` is not overlapping with `'grid'`

Does this make sense?

File: src/components/Hero/Default/index.tsx [hunk 3]
```diff
@@ -26,17 +25,27 @@ export const DefaultHero: React.FC<
...
+              .join(' ')}
+          >
             <RichText className={classes.richText} content={richText} />
-          </Cell>
+          </div>
 
           {!withoutSidebar && (
-            <Cell start={10} cols={4} startM={6} colsS={12} startS={1}>
+            <div
+              className={['cols-4 start-12 start-m-6 cols-s-8 start-s-1'].filter(Boolean).join(' ')}
```
Comment (PatrikKozak) [inline src/components/Hero/Default/index.tsx:43]: @paulpopus Are we sure `cols-4` is accurate here even with the new column count?

File: src/components/Footer/index.tsx (hunk 1 above)
Comment (paulpopus) [reply src/components/Footer/index.tsx:272]: In this particular case it's needed, the component is slightly wider than 1 col, so 2 cols keeps it aligned horizontally with the component above it

File: src/components/Hero/Default/index.tsx (hunk 3 above)
Comment (paulpopus) [reply src/components/Hero/Default/index.tsx:43]: Yeah this is ok in my testing, the page width is wider, so a side-by-side comparison of say /case-studies this is pretty close not accounting for typo-related issues

**Pull request merged**